pkgs/applications/networking/cluster/kube3d/default.nix

```diff
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "kube3d";
-  version = "3.0.2";
-  k3sVersion = "1.18.6-k3s1";
+  version = "3.1.5";
+  k3sVersion = "1.18.9-k3s1";
```

Comment:
Latest k3s version is 1.18.10, should we use that one?

Reply:
I've just been copying whatever value is in the released k3d binary, the image can be overriden with the `--image` flag when creating the cluster

Reply:
Fair enough, makes sense to align with the upstream version